h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -49,18 +49,17 @@ class Annotator.Plugin.Toolbar extends Annotator.Plugin
           state = not @annotator.visibleHighlights
           @annotator.setVisibleHighlights state
     ,
-      "title": "Highlighting Mode"
-      "class": "h-icon-highlighter"
-      "on":
-        "click": (event) =>
-          event.preventDefault()
-          event.stopPropagation()
-          state = not (@annotator.tool is 'highlight')
-          tool = if state then 'highlight' else 'comment'
-          @annotator.setTool tool
-    ,
+    ## TODO: if we are going to allow highlighting on mobile we should show this button on a selection (mobile only).
```

Comment:
I'm going to do it for you this time, because I'm about to merge this, but please don't leave commented code in your PRs. If something isn't used, delete it. This is what version control is for.